README.md

```diff
@@ -14,6 +14,7 @@ What: Come and try new tea everyday
 
 | #  | Brand              | Type        | Name                        | Origin      | Tea Master      | Date     |
 |----|--------------------|-------------|-----------------------------|-------------|-----------------|----------|
+| 57 | [T2                | [White]     | [White Ginger]              | -           | Jesse Smit      | 21-09-16 |
```

Comment:
Brand - [T2]
Name - [White Ginger] - don't forget to add it in the links below ;-)